pkg/local_workflows/output_workflow_test.go

```diff
@@ -194,6 +194,21 @@ func Test_Output_outputWorkflowEntryPoint(t *testing.T) {
 		assert.Equal(t, 1, len(output))
 	})
 
+	t.Run("should not output anything for content_types.LOCAL_FINDING_MODEL", func(t *testing.T) {
```

Comment:
I'm curious about this... is this something intentional, or is it a test asserting current behavior that is likely to change later?

How would the local findings model get rendered into a template, for example?

Reply:
As above this work we be done in a future update. This check is to ensure we can safely merge these changes ahead of the follow up work to render the localFindings. https://snyksec.atlassian.net/browse/CLI-506